bitnami/discourse/templates/configmaps.yaml

```diff
@@ -10,20 +10,24 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 data:
-  {{- $port:=.Values.service.port | toString }}
-  DISCOURSE_HOSTNAME: "{{ include "discourse.host" . }}{{- if ne $port "80" }}:{{ .Values.service.port }}{{ end }}"
+  {{- $port := .Values.service.port | toString }}
+  DISCOURSE_HOST: "{{ include "discourse.host" . }}{{- if ne $port "80" }}:{{ .Values.service.port }}{{ end }}"
+  DISCOURSE_SKIP_INSTALL: {{ ternary "yes" "no" .Values.discourse.skipInstall | quote }}
+  DISCOURSE_SITE_NAME: {{ .Values.discourse.siteName | quote }}
   DISCOURSE_USERNAME: {{ .Values.discourse.username | quote }}
   DISCOURSE_EMAIL: {{ .Values.discourse.email | quote }}
-  DISCOURSE_SITENAME: {{ .Values.discourse.siteName | quote }}
-  REDIS_HOST: {{ template "discourse.redisHost" . }}
-  REDIS_PORT_NUMBER: {{ template "discourse.redisPort" . }}
-  POSTGRESQL_HOST: {{ template "discourse.databaseHost" . }}
-  POSTGRESQL_PORT_NUMBER: {{ template "discourse.databasePort" . }}
-  DISCOURSE_POSTGRESQL_NAME: {{ template "discourse.databaseName" . }}
-  DISCOURSE_POSTGRESQL_USERNAME: {{ template "discourse.databaseUser" . }}
-  POSTGRESQL_ROOT_USER: "postgres"
-  {{- if .Values.discourse.skipInstall }}
```

Comment:
If now this value is not used you need to remove it from the values and README.

Reply:
It is used, I just moved it above:

```yaml
DISCOURSE_SKIP_INSTALL: {{ ternary "yes" "no" .Values.discourse.skipInstall | quote }}
```